runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -964,6 +965,14 @@ ClassFileOracle::walkMethodMethodParametersAttribute(U_16 methodIndex)
 
 }
 
+void
+ClassFileOracle::throwGenericErrorCustom(UDATA code, UDATA offset)
+{
+	PORT_ACCESS_FROM_PORT(_context->portLibrary());
+	_context->javaVM()->dynamicLoadBuffers->classFileError = (U_8 *)j9mem_allocate_memory(sizeof(J9CfrError), J9MEM_CATEGORY_CLASSES);
```

Comment:
Please add code to deal with the out-of-memory case when `j9mem_allocate_memory `returns NULL to `classFileError `.

Reply:
Thanks Cheng. I've added a check for this.